### DISCUSSION TITLE
Is there a way to change the default tutorial video urls?

### DISCUSSION THREAD

**Discussion by OfficialAndyX** [GriefPrevention/GriefPrevention#1367] [Q&A] [answered]
I wonder if i can change the youtube tutorial link in the book to one of my own?

**Comment by Jikoo** [comment, marked as answer]
In your messages.yml, replace the placeholder for the URL in that entry with your own link.

**Reply by OfficialAndyX** [reply]
I couldn't find the bit.ly link in the config, do you mean replacing the {0} in:

  SurvivalBasicsVideo2:
    Text: 'Click for Land Claim Help: {0}'
    Notes: '{0}: video URL'

**Reply by Jikoo** [reply]
Yes. The {0} is a placeholder that GP will replace. Don't include it, put your own link in its place.

**Reply by OfficialAndyX** [reply]
Didnt seem to work, does /gpreload reload those messages.yml values?

**Reply by Jikoo** [reply]
Nope. You need to restart your server for message changes to take effect.

**Reply by OfficialAndyX** [reply]
Thanks allot!
